mantle/platform/qemu.go

```diff
@@ -853,6 +853,19 @@ func (builder *QemuBuilder) VirtioChannelRead(name string) (*os.File, error) {
 	return r, nil
 }
 
+// SerialPipe reads the serial console output into a pipe
+func (builder *QemuBuilder) SerialPipe() (*os.File, error) {
```

Comment:
This is no longer used in this PR but I kept it because it'll surely be useful later.